StateTimeline / StatusHistory: Add axis visibility and width controls (#98548)
grafana/grafana: docs/sources/shared/visualizations/axis-options-size.md

---
title: Axis options
comments: |

>   This is for visualizations that only have the size property of axis. Other visualizations should use axis-options-all.

Reviewer: ```suggestion
  This file is used in the following visualizations: state timeline, status history.
```

---

> The option under the **Axis** section controls how the y-axis is rendered. It may not take effect until you click outside of the field option box you're editing. You can also press `Enter`.
> 
> | Option | Description                                                                                                                                                                                                                                                                                                                                                                      |
> | ------ | -------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
> | Width  | Set a fixed width of the axis. By default, Grafana dynamically calculates the width of an axis. By setting the width of the axis, data with different axes types can share the same display proportions. This setting makes it easier for you to compare more than one graph’s worth of data because the axes aren't shifted or stretched within visual proximity to each other. |

Reviewer: We don't need to use a table for one option, just body text is fine. Another thing, it seems like everything after "By setting the width..." is an explanation for why we added this feature, not what the user needs to know to set it. I think most of that content probably belongs in a What's new entry and not here. Here, it seems like all the user is getting is the option to control that width.

```suggestion
The **Width** option under the **Axis** section lets you set a fixed axis width. By default, Grafana dynamically calculates the width of an axis. Your changes might not take effect until you click outside of the field option box you're editing. You can also press `Enter`.
```

Author: I can move it out of a table but the text is copied from https://grafana.com/docs/grafana/latest/panels-visualizations/visualizations/time-series/#axis-options and changed to be more generic.

Reviewer: Ah gotcha. I thought this was net new text. Yeah, I don't think it needs to be there. Do you?
